# input
float[] cmd
float speed
float acc
--- 
# output
int result